File: Ds/207.cpp
#include <iostream>
using namespace std;

int a, b;
int lcm = 0, gcd = 0;
int main() {
	cin >> a >> b;
	int i = max(a, b);
	while(!lcm) {
		if (i%a == 0 && i%b==0) {
			lcm = i;
		}
		i++;
	}
	for (int i = max(a,b); i > 0; i--) {
		if (a%i == 0 && b%i==0) {
			gcd = i;
			break;
		}
	}
	cout << gcd << endl;
	cout << lcm << endl;
}

File: Ds/207.py
a, b = map(int, input().split())
gcd = 0
lcm = 0
i = max(a, b)
while(lcm == 0):
	if (i % a == 0 and i % b == 0): lcm = i
	i += 1
for i in range(max(a, b), 0, -1):
	if (a % i == 0 and b % i == 0): 
		gcd = i
		break

print(gcd)
print(lcm)